### PR TITLE
Fix cortex bug from shell.

### PR DESCRIPTION
**Pull request by joeljpoulin**
Removes a line of code that was causing the call to Cortex from the command line to not work.

### CODE REVIEW

File: cortex/built_ins/models/classifier.py [hunk 1]
```diff
@@ -133,8 +133,7 @@ class ImageClassification(SimpleClassifier):
     defaults = dict(
         data=dict(batch_size=128, inputs=dict(inputs='images')),
         optimizer=dict(optimizer='Adam', learning_rate=1e-3),
-        train=dict(epochs=200, save_on_best='losses.classifier'),
-        classifier_args=dict(dropout=0.2))
```
Comment (rdevon) [inline cortex/built_ins/models/classifier.py:137]: So this needs to be changed to `model=dict(dict(dropout=0.2))` after `train`

Comment (joeljpoulin) [reply cortex/built_ins/models/classifier.py]: Added in commit 4ea8bae.
